srcpkgs/vifm/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'vifm'
 pkgname=vifm
-version=0.12
+version=0.13
 revision=1
 build_style=gnu-configure
 configure_args="--without-gtk"
-hostmakedepends="mdocml"
+hostmakedepends="mdocml perl coreutils"
 makedepends="ncurses-devel file-devel"
 depends="desktop-file-utils"
-short_desc="Ncurses-based file manager with vi-like keybindings"
+short_desc="File manager with curses interface and a Vim-like environment"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://vifm.info/"
-distfiles="${SOURCEFORGE_SITE}/vifm/vifm-${version}.tar.bz2"
-checksum=33a9618f32b35b5b8c64483884f9ad09963ca8465b2935def79159028e27b2c0
+distfiles="https://github.com/vifm/vifm/releases/download/v${version}/vifm-${version}.tar.bz2"
+checksum=0d9293749a794076ade967ecdc47d141d85e450370594765391bdf1a9bd45075
 
-CFLAGS="-fcommon"
+CFLAGS="-fcommon -Wno-unused-result"
```

Comment:
Although I can't get the tests to pass (even when removing the conflicting Changelog text) with your patch, I'm pretty sure this change is unnecessary because the only failing test is not a complaint about these compiler warnings.

Reply:
> Although I can't get the tests to pass (even when removing the conflicting Changelog text) with your patch, I'm pretty sure this change is unnecessary because the only failing test is not a complaint about these compiler warnings.

Yes I saw that the patch from the git history was non compatible. I'll add back the original with some kind of source link back to GitHub.

The unused warning thing was just my attempt to have a clean build. 

The tests pass on alpine:latest which uses a recent musl release. Did you build on native void linux? Is that using the latest musl?